### PR TITLE
K8SPXC-630 fix cluster deletion

### PR DESCRIPTION
**Pull request by hors**
[K8SPXC-630](https://jira.percona.com/browse/K8SPXC-630) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -8,7 +8,6 @@ void CreateCluster(String CLUSTER_PREFIX) {
             source $HOME/google-cloud-sdk/path.bash.inc
             gcloud auth activate-service-account --key-file $CLIENT_SECRET_FILE
             gcloud config set project $GCP_PROJECT
-            gcloud container clusters list --filter $CLUSTER_NAME-${CLUSTER_PREFIX} --zone $GKERegion --format='csv[no-heading](name)' | xargs gcloud container clusters delete --zone $GKERegion --quiet || true
```
Comment (delgod) [inline Jenkinsfile:11]: can we keep this line? should reduce number of fails

**Review by delgod — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (hors) [reply Jenkinsfile:11]: yes, we can but ...
 you do not trust me ? :) or even worse for my code ))))

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| big-data | passed |
| scheduled-backup | passed |
| recreate | passed |
| upgrade-haproxy | passed |
| storage | passed |
| scaling | passed |
| haproxy | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| demand-backup | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/879/commits/9323b949eb1ccf4c7701b455ec7bb454bb14bba2
image: `perconalab/percona-xtradb-cluster-operator:PR-879-9323b949`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| recreate | passed |
| scheduled-backup | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| haproxy | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/879/commits/3fdbe39f1416ce734387a10fee503ed70f95f26c
image: `perconalab/percona-xtradb-cluster-operator:PR-879-3fdbe39f`

**Pull request merged**